iterative/aws/provider.go

```diff
@@ -336,7 +336,29 @@ func awsClient(region string) (*ec2.EC2, error) {
 }
 
 //ImageRegions provider available image regions
-var ImageRegions = []string{"us-east-1", "us-east-2", "us-west-1", "us-west-2", "eu-north-1", "eu-west-1", "eu-west-2"}
+var ImageRegions = []string{
+	"us-east-2",
+	"us-east-1",
+	"us-west-1",
+	"us-west-2",
+	"af-south-1",
+	"ap-east-1",
+	"ap-south-1",
+	"ap-northeast-3",
+	"ap-northeast-2",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"ap-northeast-1",
+	"ca-central-1",
+	"eu-central-1",
+	"eu-west-1",
+	"eu-west-2",
+	"eu-south-1",
+	"eu-west-3",
+	"eu-north-1",
+	"me-south-1",
+	"sa-east-1",
```

Comment:
```suggestion
	"us-east-2",
	"us-east-1",
	"us-west-1",
	"us-west-2",
	"ap-south-1",
	"ap-northeast-3",
	"ap-northeast-2",
	"ap-southeast-1",
	"ap-southeast-2",
	"ap-northeast-1",
	"ca-central-1",
	"eu-central-1",
	"eu-west-1",
	"eu-west-2",
	"eu-west-3",
	"eu-north-1",
	"sa-east-1",
```

**Conflicting regions**

* af-south-1	
* ap-east-1
* eu-south-1
* me-south-1

@JIoJIaJIu, may we opt-in to these regions for deploying the CML machine image?

Reply:
@0x2b3bfa0 can we merge this or we still have those regions with issues?

Reply:
@DavidGOrtega, please let me try again, I don't know if the missing regions have been enabled.

Reply:
@DavidGOrtega & @JIoJIaJIu, it looks like they haven't been opted-in yet. 🤔